jest.config.ts

```diff
@@ -0,0 +1,31 @@
+const esmModules = [
```

Comment:
Why is this needed? We don't use any of these libs in the internals of the library.

Reply:
This is recommended configuration introduced by expo. 

https://docs.expo.dev/guides/testing-with-jest/#configuration

If we use pnpm,  the module resolution system is a bit different from npm and the modules cache node_modules/.pnpm, so need manually setting at least, '@react-native/' namespace. See [react-native preset](https://github.com/facebook/react-native/blob/main/jest.config.js#L34).

It's not high possibility to use another native-library, but it's ready for testing with third party libraries. If you don't need them, I remove them except for first react-native configuration which needs transpiled.